src/components/organisms/ChatSidebar/ChatSidebar.scss

```
@@ -8,10 +8,12 @@ $header-height: 45px;
   position: fixed;
   display: flex;
   flex-flow: column;
-  top: 66px;
+  top: $navbar-height;
   right: 0;
-  width: 360px;
-  height: calc(100vh - 96px);
+  bottom: 0;
+  width: $sidebar-width;
+  min-width: $sidebar-width;
+  min-height: calc(100% - #{$navbar-height + $footer-height});
```

Since we're already using `calc` here, we may as well show our full 'calculations'. Will make the 'why' of it more obvious when viewed in devtools/etc
```suggestion
  min-height: calc(100% - #{$navbar-height} - #{$footer-height});
```